pkg/controller/scyllacluster/sync.go

```diff
@@ -263,6 +362,18 @@ func (scc *Controller) sync(ctx context.Context, key string) error {
 		// TODO: Set degraded condition
 	}
 
+	err = scc.syncServiceAccounts(ctx, sc, serviceAccounts)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("can't sync serviceaccounts: %w", err))
+		// TODO: Set degraded condition
+	}
+
+	err = scc.syncRoleBindings(ctx, sc, roleBindings)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("can't sync rolebindings: %w", err))
+		// TODO: Set degraded condition
+	}
+
```

Comment:
Shouldn't these be above statefulset? It's a dependency of it

Reply:
I don't think these are a hard dependency but I can move them